docker/image_types.yaml

```diff
@@ -2,34 +2,44 @@
 # dependency graph.
 
 'base-image':
+  # base: "True"
```

Comment:
Don't know what this commented out stuff is supposed to be doing.

Reply:
Resolved.